flink-cdc-common/pom.xml

```diff
@@ -41,6 +41,14 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
```

Comment:
is it necessary or added by accident?

Reply:
Should be an unintentional change, reverted.